inst/extdata/tissue_codes.R

```diff
@@ -2,6 +2,7 @@
 # R script to generate the tissue codes and colors
 library(data.table)
 library(devtools)
+library(MotrpacBicQC)
```

Comment:
usually better to use `devtools::load_all()` than loading the library if you are changing data in the library

Reply:
You are right. And that's how I've always done it but I decided this time to load the library instead in that way. I will go back. Thanks for pointing this out